modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/core/actions/eventTypes.es.js

```diff
@@ -45,6 +45,10 @@ const SECTION = {
 	ADD: 'section_add',
 };
 
+const ELEMENT_SET = {
+	ADD: 'element_set_add',
+};
+
```

Comment:
It would be great if we could remove this from the `core` since it is a Forms use case, maybe something to add to future research to explore a way to add different types of DnD without keeping the specific code inside the `useDrop`.